python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/CODE_LOCATION_NAME_PLACEHOLDER/definitions.py

```diff
@@ -1,6 +1,6 @@
 from dagster import Definitions, load_assets_from_modules
 
-from CODE_LOCATION_NAME_PLACEHOLDER import assets  # type: ignore
+from . import assets
```

Comment:
I think we should discourage relative imports and promote the usage of absolute imports instead

Reply:
Oh I see. I remember doing something like `project_name.lower().replaces("-", "_")` in my cookiecutter setup for the same reason. 

I guess since is't just a quickstart we don't really care here.